tex/gregoriotex-main.tex

```diff
@@ -1644,15 +1644,18 @@
 \newif\ifgre@in@euouae
 \gre@in@euouaefalse
 
+\newif\ifgre@firstin@euouae
+\gre@in@euouaefalse
```

Comment:
As is this line is a repeat of 1645.  I think you meant `\gre@firstin@euouaefalse`